src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -184,57 +222,164 @@ def _execute(self, rseList):
             self.plineCounters[pline.name]['rsesCleaned'], \
             self.plineCounters[pline.name]['deletedSuccess']
 
+    # @profile
     def cleanRSE(self, rse):
         """
         The method to implement the actual deletion of files for an RSE.
         :param rse: MSUnmergedRSE object to be cleaned
         :return:    The MSUnmergedRSE object
         """
-        # if self.msConfig['enableRealMode']:
+
+        # Create the gfal2 context object:
         try:
-            # for fileUnmerged in rse['files']['toDelete']:
-            #     try:
-            #         self.gfalCommand(rse['delInterface'], fileUnmerged)
-            #         rse['counters']['numFilesDeleted'] += 1
-            #         rse['files']['deletedSuccess'].append(fileUnmerged)
-            #     except Exception as ex:
-            #         rse['files']['deletedFail'].append(fileUnmerged)
-            #         msg = "Error while trying to delete file: %s for RSE: %s"
-            #         msg += "Will retry in the next cycle. Err: %s"
-            #         self.logger.debug(msg, fileUnmerged, rse['name'], str(ex))
-            rse['isClean'] = self._checkClean(rse)
+            ctx = gfal2.creat_context()
```

Comment:
I don't know anything about this gfal library. Is this a typo?

Reply:
nope:) it is not a typo- it is how it is.